app/javascript/src/components/Dashboard/Notes/NoteTable.jsx

```diff
@@ -26,15 +27,20 @@ export default function NoteTable({
                 }}
               />
             </th>
-            <th className="text-left">Title</th>
-            <th className="text-left">Description</th>
+            <th className="text-left text-gray-400">Title</th>
+            <th className="text-left text-gray-400">Description</th>
+            <th className="text-center text-gray-400">Tags</th>
+            <th className="text-center text-gray-400">Created Date</th>
+            <th className="text-center text-gray-400">Due Date</th>
+            <th className="text-center text-gray-400">Contact</th>
+            <th></th>
           </tr>
         </thead>
         <tbody>
           {notes.map(note => (
             <tr
               key={note.id}
-              className={"cursor-pointer bg-white hover:bg-gray-50"}
+              className={"group cursor-pointer bg-white hover:bg-gray-50"}
```

Comment:
Please remove this since we are adding `nui-table--hover` class for hover effects.